babi/file.py

```diff
@@ -707,7 +707,10 @@ def _comment_remove(self, lineno: int, prefix: str) -> None:
     def _comment_add(self, lineno: int, prefix: str, s_offset: int) -> None:
         line = self.buf[lineno]
 
-        self.buf[lineno] = f'{line[:s_offset]}{prefix}{line[s_offset:]}'
+        if line == '':
+             self.buf[lineno] = f'{line[:s_offset]}{prefix}{line[s_offset:]}'
```

Comment:
I would write:

```python
if not line:
     self.buf[lineno] = prefix
else:
    ...
```

Reply:
Ah is that why it doesn't pass the 100% coverage?

Reply:
yeah, otherwise I'd be asserting the bug under test 😆

Reply:
or, at least, the `if not line` isn't the reason -- the reason is there's no test for the change yet!

Reply:
I see! I wrote a test. Seeing if this will fix the coverage.